auto-value-gson/src/test/java/com/ryanharter/auto/value/gson/AutoValueGsonExtensionTest.java

```diff
@@ -34,8 +33,7 @@ public void setup() {
         + "}");
   }
 
```

Comment:
For more comprehensive testing, I took the optional property tests out of the `simple()` method and instead created entirely new tests based on `simple()` in which all properties are optional. Java and Guava optionals are tested in separate methods (with Java including the primitive optional wrappers), and each of Java/Guava have separate tests for w/ and w/o default setter cases.